src/platform/plugins/shared/dashboard/public/dashboard_api/panels_manager.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Rename to `layout_manager.test.ts`. Also, mind updating with main. `initializePanelsManager` is now called `initializeLayoutManager`